### PR TITLE
Detect terminated employee and disable them

### PR DESCRIPTION
**Pull request by PChouzenoux**
New cronjob project that detect employee that gave access to Studiobox for a loyalty account. Then disable this access, and publish update for the brain to be able to blacklist them.

Update for brain is a file added in a s3 bucket and folder.

### CODE REVIEW

File: config/default.js [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+      path: 'data/customer_profiles/blacklist',
+    },
+  },
+  auth: {
+    graphUrl: 'https://graph.microsoft.com/v1.0',
+    url: 'https://login.microsoftonline.com/corp.gdglobal.ca/oauth2/v2.0',
+    permissionScope: 'user.read.all',
+    domain: 'corp.gdglobal.ca',
+    clientId: 'd9f5ce79-5444-4f7c-81e8-5d24186d9949',
+    clientSecret: 'f4KcV/d+PLO]go.vLE43A8YvrP5GiAv[',
```
Comment (joeleb01) [inline config/default.js:22]: DANGER, DANGER, DANGER!! :P

File: src/customerAccess/CustomerAccessStore.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        }
+        cb();
+      },
+    });
+
+    return this.datastore
+      .createQuery(this.customerAccessKind)
+      .filter('hasAccess', '=', true)
+      .runStream()
+      .pipe(transform);
```
Comment (joeleb01) [inline src/customerAccess/CustomerAccessStore.ts:38]: NIce!! :)

File: config/default.js (hunk 1 above)
Comment (PChouzenoux) [reply config/default.js:22]: oups ! I gonna fix it

File: src/customerAccess/CustomerAccessStore.ts (hunk 2 above)
Comment (PChouzenoux) [reply src/customerAccess/CustomerAccessStore.ts:38]: Yes a lot of stream in the service !

**Pull request merged**